src/SDK/SmartDetectorsBuildTask/build/net461/Microsoft.Azure.Monitoring.SmartDetectors.Build.targets

```diff
@@ -2,12 +2,15 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <RunPostBuildEvent>Always</RunPostBuildEvent>
+    <SmartDetectorsTaskPackagePath Condition="Exists('$(OutDir)')">$(OutDir)</SmartDetectorsTaskPackagePath>
+    <SmartDetectorsTaskPackagePath Condition="!Exists('$(OutDir)')">$(ProjectDir)$(OutDir)</SmartDetectorsTaskPackagePath>
```

Comment:
>!Exists('$(OutDir)') [](start = 46, length = 20)

not so sure this is the correct approach - what if OutDir already exists but it's relative?

Reply:
Improved the condition

---
In reply to: [254192813](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/47#discussion_r254192813) [](ancestors = 254192813)